CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/shaders/PassThru.vs

```diff
@@ -1,17 +1,19 @@
-#version 330 core
+#version 450
 
 
 // === UNIFORMS
-layout(std140) uniform UniformBlock {
-    mat4 mvpMatrix;
-};
+layout(std140, binding = 0) uniform UniformBlock {
+    mat4 mvMatrix;
```

Comment:
Why did you rename this?  Given there is no project to screen space happening in the geometry shader I think this might well be a model-view-projection matrix.

Reply:
renamed back to mvp